### PR TITLE
HTTPS support

### PR DESCRIPTION
**Pull request by dimon222**
Fix for #19 

The only thing is - do we need to apply any corrections in hadoop_conf detection to pull HTTPS urls out as well, or no?

~~PS: this will conflict with #26, might require rebase incase of merge needs~~

### CODE REVIEW

**Comment by dimon222** [comment]
@lresende any success?

**Comment by dimon222** [comment]
@lresende since my refactoring PR got merged, I recreated this PR to make it compatible.

File: yarn_api_client/application_master.py [hunk 1]
```diff
@@ -19,13 +19,15 @@ class ApplicationMaster(BaseYarnAPI):
     :param int port: Proxy HTTP port
     :param int timeout: API connection timeout in seconds
     :param boolean kerberos_enabled: Flag identifying is Kerberos Security has been enabled for YARN
+    :param boolean https: Flag identifying HTTPS is used instead of HTTP for YARN
+    :param boolean https_verify: Flag identifying SSL certificate should be ignored in case of HTTPS
```
Comment (lresende) [inline yarn_api_client/application_master.py]: As we are forwarding this parameter all way to `requests`, could you please use/append similar documentation as the one available in requests: `Either a boolean, in which case it controls whether we verify the server’s TLS certificate, or a string, in which case it must be a path to a CA bundle to use`

**Comment by lresende** [comment]
@dimon222 Could you please rebase this as well.

**Comment by dimon222** [comment]
Closed, better approach is suggested there - https://github.com/toidi/hadoop-yarn-api-python-client/issues/33

**Pull request closed without merging**